x-pack/platform/plugins/shared/ml/server/models/data_recognizer/modules/security_azure_activitylogs/ml/azure_activitylogs_rare_event_action_for_a_city.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "Security: Azure Activity Logs - Looks for Azure activity event actions that, while not inherently suspicious or abnormal, are sourcing from a geolocation (city) that is unusual. This can be the result of compromised credentials or keys.",
+  "groups": ["security", "azure"],
+  "analysis_config": {
+    "bucket_span": "60m",
+    "detectors": [
+      {
+        "detector_description": "Detects rare event actions for a city.",
```

Comment:
Suggest adding more details like what you have on `x-pack/platform/plugins/shared/ml/server/models/data_recognizer/modules/security_azure_activitylogs/ml/azure_activitylogs_rare_event_action_on_failure.json`